command/server.go

```diff
@@ -1106,6 +1106,15 @@ func (c *ServerCommand) Run(args []string) int {
 
 	logProxyEnvironmentVariables(c.logger)
 
+   	if clusterName := os.Getenv("VAULT_CLUSTER_NAME"); clusterName != "" {
+		var err error
+		config.ClusterName = clusterName
+		if err != nil {
```

Comment:
I'm guessing this whole if statement can be omitted, since err is never assigned to anything, so it will always be nil.

Reply:
Whoops that's what I get for copying an env with with Parsebool 😅

Reply:
Your commit to remove the error check didn't actually remove it. 😅